content/plugins/aws.md

```diff
@@ -46,12 +53,30 @@ All cloud resource nodes have common properties:
 **Properties**
 
   * `use_external_resource` a boolean for setting whether to create the resource or use an existing one. See the [using existing resources section](#using-existing-resources). Defaults to `false`.
-  * `resource_id` this can be the name of a resource when one is supported (security group and keypair) or the ID of an existing resource when the `use_external_resource` property is set to `true` (see the [using existing resources section](#using-existing-resources)). Defaults to `''` (empty string).
+  * `resource_id` The ID of an existing resource when the `use_external_resource` property is set to `true` (see the [using existing resources section](#using-existing-resources)). Defaults to `''` (empty string). When using a new (not external) security group or key pair, this will become the resource name.
   * `aws_config` a dictionary that contains values you would like to pass to the connection client. For information on values that are accepted, please see [boto documentation](http://boto.readthedocs.org/en/latest/ref/ec2.html#boto.ec2.connection.EC2Connection)
 
+Every time you manage a resource with Cloudify, we create one or more clients with AWS API. You specify the configuration for these clients using the aws_config property. It should be a dictionary, with the following values:
+
+Your AWS API access credentials. [Read more](http://docs.aws.amazon.com/AWSSecurityCredentials/1.0/AboutAWSCredentials.html#).
+
+  * AWS_ACCESS_KEY_ID (required)
+  * AWS_ACCESS_SECRET_KEY (required)
+
+Region information:
+
+  * ec2_region_name (required, except with the cloudify.aws.nodes.ElasticLoadBalancer node type.) This is the EC2 region name, such as 'us-east-1'. You may also use the word 'region' to refer to the same thing
```

Comment:
use backticks to highlight ec2_region_name and any other part of the docs that is a reference to an actual code or node type name